PancakeSwap-Bots/Lottery-Admin-Operations/src/findings.ts

```diff
@@ -0,0 +1,32 @@
+import { Finding, FindingType, FindingSeverity } from "forta-agent";
+import { ALERTS } from "./abi";
+
+export const createEventFinding = (_name: string, _description: string, _metadata: {}) => {
+  return Finding.fromObject({
+    name: `Event: ${_name === "NewRandomGenerator" ? "NewRandomGenerator" : _name}`,
```

Comment:
```suggestion
    name: `Event: ${_name}`,
```
This is the same, right?

I would use a slightly more descriptive/general name:
` PanckaSwap event emitted`
without using the event name, so that will be part of the description

Reply:
yes, it was some leftover logic that I forgot to delete